package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "sku",
   "version": "0.0.0-development",
-  "description": "Front-end development toolkit, powered by Webpack, Babel, CSS Modules, Less and Jest",
+  "description":
```

Comment:
It looks like you didn't change this file, and it's just been formatted with Prettier—is that right?

Reply:
I'm not sure we want to format package.json with prettier, since npm already enforces its own formatting via JSON.stringify when installing packages.

Reply:
Whoops, `formatOnSave`!